libs/sdk-core/src/input_parser.rs

```diff
@@ -374,6 +375,14 @@ fn lnurl_decode(encoded: &str) -> LnUrlResult<(String, String, Option<String>)>
     }
 }
 
+/// Creates an HTTP client with a built-in connection timeout
+pub(crate) fn get_reqwest_client() -> Result<reqwest::Client> {
+    reqwest::Client::builder()
+        .timeout(Duration::from_secs(60))
```

Comment:
I would use a lower number here, I suggest 30 seconds as we have in breez mobile.

Reply:
Done